app/models/declaration/Transport.scala

```diff
@@ -55,4 +55,6 @@ case class Transport(
 
 object Transport {
   implicit val format: Format[Transport] = Json.format[Transport]
+
+  val optionNone = "option_none"
```

Comment:
Could this be moved to `TransportDetails`? To me it is more related to that scope. Unless there is something I don't understand...

Reply:
I thought it might be useful for if/when other modes of transport need a "none" option.  E.g. the ticket you have picked up.

Reply:
My second thought is that since we are sharing this value between two services, maybe we should have a dedicated type? There is "Null object (anti) pattern" and I was thinking about using something similar here. Strings are very susceptible to type errors.